Tests/Util/FilenameUtilsTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Vich\UploaderBundle\Tests\Util;
+
+use PHPUnit\Framework\TestCase;
+use Vich\UploaderBundle\Util\FilenameUtils;
+
+class FilenameUtilsTest extends TestCase
+{
+    /**
+     * @dataProvider spitNameByExtensionProvider
+     */
+    public function spitNameByExtension($filename, $basename, $extension)
+    {
+        $this->assertSame([$basename, $extension], FilenameUtils::spitNameByExtension($filename));
+    }
+
+    public function spitNameByExtensionProvider()
+    {
+        return [
+            ['Текстовый файл.txt', 'Текстовый файл', 'txt'],
```

Comment:
And that's a shame, because if a test is failing you don't get any clue about what is failing